### ISSUE TITLE
2025-02-27, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#96]
# Today's Arxiv Papers

(1) [Norm Growth and Stability Challenges in Localized Sequential Knowledge Editing](http://arxiv.org/abs/2502.19416v1), 2025-02-26

(2) [Project Alexandria: Towards Freeing Scientific Knowledge from Copyright Burdens via LLMs](http://arxiv.org/abs/2502.19413v1), 2025-02-26

(3) [Code to Think, Think to Code: A Survey on Code-Enhanced Reasoning and Reasoning-Driven Code Intelligence in LLMs](http://arxiv.org/abs/2502.19411v1), 2025-02-26

(4) [Less or More: Towards Glanceable Explanations for LLM Recommendations Using Ultra-Small Devices](http://arxiv.org/abs/2502.19410v1), 2025-02-26

(5) [ImageChain: Advancing Sequential Image-to-Text Reasoning in Multimodal Large Language Models](http://arxiv.org/abs/2502.19409v1), 2025-02-26

(6) [Learning Code-Edit Embedding to Model Student Debugging Behavior](http://arxiv.org/abs/2502.19407v1), 2025-02-26

(7) [Verde: Verification via Refereed Delegation for Machine Learning Programs](http://arxiv.org/abs/2502.19405v1), 2025-02-26

(8) [General Reasoning Requires Learning to Reason from the Get-go](http://arxiv.org/abs/2502.19402v1), 2025-02-26

(9) [TheoremExplainAgent: Towards Multimodal Explanations for LLM Theorem Understanding](http://arxiv.org/abs/2502.19400v1), 2025-02-26

(10) [HDEE: Heterogeneous Domain Expert Ensemble](http://arxiv.org/abs/2502.19385v1), 2025-02-26

(11) [DataMan: Data Manager for Pre-training Large Language Models](http://arxiv.org/abs/2502.19363v1), 2025-02-26

(12) [Can Large Language Models Detect Errors in Long Chain-of-Thought Reasoning?](http://arxiv.org/abs/2502.19361v1), 2025-02-26

(13) [Controlled Diversity: Length-optimized Natural Language Generation](http://arxiv.org/abs/2502.19347v1), 2025-02-26

(14) [Evaluating LLMs and Pre-trained Models for Text Summarization Across Diverse Datasets](http://arxiv.org/abs/2502.19339v1), 2025-02-26

(15) [Agentic Reward Modeling: Integrating Human Preferences with Verifiable Correctness Signals for Reliable Reward Systems](http://arxiv.org/abs/2502.19328v1), 2025-02-26

(16) [Shh, don't say that! Domain Certification in LLMs](http://arxiv.org/abs/2502.19320v1), 2025-02-26

(17) [FSPO: Few-Shot Preference Optimization of Synthetic Preference Data in LLMs Elicits Effective Personalization to Real Users](http://arxiv.org/abs/2502.19312v1), 2025-02-26

(18) [Rethinking LLM Unlearning Objectives: A Gradient Perspective and Go Beyond](http://arxiv.org/abs/2502.19301v1), 2025-02-26

(19) [Agent-centric Information Access](http://arxiv.org/abs/2502.19298v1), 2025-02-26

(20) [Complex LLM Planning via Automated Heuristics Discovery](http://arxiv.org/abs/2502.19295v1), 2025-02-26

(21) [Efficient Federated Search for Retrieval-Augmented Generation](http://arxiv.org/abs/2502.19280v1), 2025-02-26

(22) [ArtInsight: Enabling AI-Powered Artwork Engagement for Mixed Visual-Ability Families](http://dx.doi.org/10.1145/3708359.3712082), 2025-02-26

(23) [Two Heads Are Better Than One: Dual-Model Verbal Reflection at Inference-Time](http://arxiv.org/abs/2502.19230v1), 2025-02-26

(24) [A Hybrid Transformer Architecture with a Quantized Self-Attention Mechanism Applied to Molecular Generation](http://arxiv.org/abs/2502.19214v1), 2025-02-26

(25) [Negation-Induced Forgetting in LLMs](http://arxiv.org/abs/2502.19211v1), 2025-02-26

(26) [Bi'an: A Bilingual Benchmark and Model for Hallucination Detection in Retrieval-Augmented Generation](http://arxiv.org/abs/2502.19209v1), 2025-02-26

(27) [Simulation of Language Evolution under Regulated Social Media Platforms: A Synergistic Approach of Large Language Models and Genetic Algorithms](http://arxiv.org/abs/2502.19193v1), 2025-02-26

(28) [BIG-Bench Extra Hard](http://arxiv.org/abs/2502.19187v1), 2025-02-26

(29) [UQABench: Evaluating User Embedding for Prompting LLMs in Personalized Question Answering](http://arxiv.org/abs/2502.19178v1), 2025-02-26

(30) [MEDDxAgent: A Unified Modular Agent Framework for Explainable Automatic Differential Diagnosis](http://arxiv.org/abs/2502.19175v1), 2025-02-26

(31) [CodeIF: Benchmarking the Instruction-Following Capabilities of Large Language Models for Code Generation](http://arxiv.org/abs/2502.19166v1), 2025-02-26

(32) [TestNUC: Enhancing Test-Time Computing Approaches through Neighboring Unlabeled Data Consistency](http://arxiv.org/abs/2502.19163v1), 2025-02-26

(33) [Detecting Linguistic Indicators for Stereotype Assessment with Large Language Models](http://arxiv.org/abs/2502.19160v1), 2025-02-26

(34) [A Sliding Layer Merging Method for Efficient Depth-Wise Pruning in LLMs](http://arxiv.org/abs/2502.19159v1), 2025-02-26

(35) [When Personalization Meets Reality: A Multi-Faceted Analysis of Personalized Preference Learning](http://arxiv.org/abs/2502.19158v1), 2025-02-26

(36) [Isolating Language-Coding from Problem-Solving: Benchmarking LLMs with PseudoEval](http://arxiv.org/abs/2502.19149v1), 2025-02-26

(37) [Amulet: ReAlignment During Test Time for Personalized Preference Adaptation of LLMs](http://arxiv.org/abs/2502.19148v1), 2025-02-26

(38) [A Temporal Planning Framework for Multi-Agent Systems via LLM-Aided Knowledge Base Management](http://arxiv.org/abs/2502.19135v1), 2025-02-26

(39) [DBox: Scaffolding Algorithmic Programming Learning through Learner-LLM Co-Decomposition](http://arxiv.org/abs/2502.19133v1), 2025-02-26

(40) [Self-Memory Alignment: Mitigating Factual Hallucinations with Generalized Improvement](http://arxiv.org/abs/2502.19127v1), 2025-02-26

(41) [Evaluating Gender Bias in German Machine Translation](http://arxiv.org/abs/2502.19104v1), 2025-02-26

(42) [LongEval: A Comprehensive Analysis of Long-Text Generation Through a Plan-based Paradigm](http://arxiv.org/abs/2502.19103v1), 2025-02-26

(43) [Language-Driven Opinion Dynamics in Agent-Based Simulations with LLMs](http://arxiv.org/abs/2502.19098v1), 2025-02-26

(44) [Nexus: A Lightweight and Scalable Multi-Agent Framework for Complex Tasks Automation](http://arxiv.org/abs/2502.19091v1), 2025-02-26

(45) [Sparse Brains are Also Adaptive Brains: Cognitive-Load-Aware Dynamic Activation for LLMs](http://arxiv.org/abs/2502.19078v1), 2025-02-26

(46) [IndicEval-XL: Bridging Linguistic Diversity in Code Generation Across Indic Languages](http://arxiv.org/abs/2502.19067v1), 2025-02-26

(47) [Can Large Language Models Outperform Non-Experts in Poetry Evaluation? A Comparative Study Using the Consensual Assessment Technique](http://arxiv.org/abs/2502.19064v1), 2025-02-26

(48) [MathClean: A Benchmark for Synthetic Mathematical Data Cleaning](http://arxiv.org/abs/2502.19058v1), 2025-02-26

(49) [Beyond Surface-Level Patterns: An Essence-Driven Defense Framework Against Jailbreak Attacks in LLMs](http://arxiv.org/abs/2502.19041v1), 2025-02-26

(50) [FungalZSL: Zero-Shot Fungal Classification with Image Captioning Using a Synthetic Data Approach](http://arxiv.org/abs/2502.19038v1), 2025-02-26

(51) [Binary Neural Networks for Large Language Model: A Survey](http://arxiv.org/abs/2502.19008v1), 2025-02-26

(52) [The Sharpness Disparity Principle in Transformers for Accelerating Language Model Pre-Training](http://arxiv.org/abs/2502.19002v1), 2025-02-26

(53) [MEBench: Benchmarking Large Language Models for Cross-Document Multi-Entity Question Answering](http://arxiv.org/abs/2502.18993v1), 2025-02-26

(54) [OntologyRAG: Better and Faster Biomedical Code Mapping with Retrieval-Augmented Generation (RAG) Leveraging Ontology Knowledge Graphs and Large Language Models](http://arxiv.org/abs/2502.18992v1), 2025-02-26

(55) [GenTool: Enhancing Tool Generalization in Language Models through Zero-to-One and Weak-to-Strong Simulation](http://arxiv.org/abs/2502.18990v1), 2025-02-26

(56) [PEToolLLM: Towards Personalized Tool Learning in Large Language Models](http://arxiv.org/abs/2502.18980v1), 2025-02-26

(57) [Low-Confidence Gold: Refining Low-Confidence Samples for Efficient Instruction Tuning](http://arxiv.org/abs/2502.18978v1), 2025-02-26

(58) [Know You First and Be You Better: Modeling Human-Like User Simulators via Implicit Profiles](http://arxiv.org/abs/2502.18968v1), 2025-02-26

(59) [DualSpec: Text-to-spatial-audio Generation via Dual-Spectrogram Guided Diffusion Model](http://arxiv.org/abs/2502.18952v1), 2025-02-26

(60) [Towards Label-Only Membership Inference Attack against Pre-trained Large Language Models](http://arxiv.org/abs/2502.18943v1), 2025-02-26

(61) [MathTutorBench: A Benchmark for Measuring Open-ended Pedagogical Capabilities of LLM Tutors](http://arxiv.org/abs/2502.18940v1), 2025-02-26

(62) [JailBench: A Comprehensive Chinese Security Assessment Benchmark for Large Language Models](http://arxiv.org/abs/2502.18935v1), 2025-02-26

(63) [Kanana: Compute-efficient Bilingual Language Models](http://arxiv.org/abs/2502.18934v1), 2025-02-26

(64) [Talking like Piping and Instrumentation Diagrams (P&IDs)](http://arxiv.org/abs/2502.18928v1), 2025-02-26

(65) [ClassInvGen: Class Invariant Synthesis using Large Language Models](http://arxiv.org/abs/2502.18917v1), 2025-02-26

(66) [END: Early Noise Dropping for Efficient and Effective Context Denoising](http://arxiv.org/abs/2502.18915v1), 2025-02-26

(67) [CLLoRA: An Approach to Measure the Effects of the Context Length for LLM Fine-Tuning](http://arxiv.org/abs/2502.18910v1), 2025-02-26

(68) [An Empirical Study on Commit Message Generation using LLMs via In-Context Learning](http://arxiv.org/abs/2502.18904v1), 2025-02-26

(69) [From Hours to Minutes: Lossless Acceleration of Ultra Long Sequence Generation up to 100K Tokens](http://arxiv.org/abs/2502.18890v1), 2025-02-26

(70) [On Pruning State-Space LLMs](http://arxiv.org/abs/2502.18886v1), 2025-02-26

(71) [Letters from Future Self: Augmenting the Letter-Exchange Exercise with LLM-based Agents to Enhance Young Adults' Career Exploration](http://dx.doi.org/10.1145/3706598.3714206), 2025-02-26

(72) [Learning to Generate Structured Output with Schema Reinforcement Learning](http://arxiv.org/abs/2502.18878v1), 2025-02-26

(73) [Learning to Align Multi-Faceted Evaluation: A Unified and Robust Framework](http://arxiv.org/abs/2502.18874v1), 2025-02-26

(74) [Multi-LLM Collaborative Search for Complex Problem Solving](http://arxiv.org/abs/2502.18873v1), 2025-02-26

(75) [Sherlock: Towards Multi-scene Video Abnormal Event Extraction and Localization via a Global-local Spatial-sensitive LLM](http://arxiv.org/abs/2502.18863v1), 2025-02-26

(76) [Investigating Generalization of One-shot LLM Steering Vectors](http://arxiv.org/abs/2502.18862v1), 2025-02-26

(77) [Marking Code Without Breaking It: Code Watermarking for Detecting LLM-Generated Code](http://arxiv.org/abs/2502.18851v1), 2025-02-26

(78) [A Causal Lens for Evaluating Faithfulness Metrics](http://arxiv.org/abs/2502.18848v1), 2025-02-26

(79) [Sliding Window Attention Training for Efficient Large Language Models](http://arxiv.org/abs/2502.18845v1), 2025-02-26

(80) [REALM-Bench: A Real-World Planning Benchmark for LLMs and Multi-Agent Systems](http://arxiv.org/abs/2502.18836v1), 2025-02-26

(81) [Evidence-Driven Marker Extraction for Social Media Suicide Risk Detection](http://arxiv.org/abs/2502.18823v1), 2025-02-26

(82) [Data-Efficient Multi-Agent Spatial Planning with LLMs](http://arxiv.org/abs/2502.18822v1), 2025-02-26

(83) [CAMEx: Curvature-aware Merging of Experts](http://arxiv.org/abs/2502.18821v1), 2025-02-26

(84) [Judge as A Judge: Improving the Evaluation of Retrieval-Augmented Generation through the Judge-Consistency of Large Language Models](http://arxiv.org/abs/2502.18817v1), 2025-02-26

(85) [Holistic Audit Dataset Generation for LLM Unlearning via Knowledge Graph Traversal and Redundancy Removal](http://arxiv.org/abs/2502.18810v1), 2025-02-26

(86) [ANPMI: Assessing the True Comprehension Capabilities of LLMs for Multiple Choice Questions](http://arxiv.org/abs/2502.18798v1), 2025-02-26

(87) [Anything Goes? A Crosslinguistic Study of (Im)possible Language Learning in LMs](http://arxiv.org/abs/2502.18795v1), 2025-02-26

(88) [SolEval: Benchmarking Large Language Models for Repository-level Solidity Code Generation](http://arxiv.org/abs/2502.18793v1), 2025-02-26

(89) [Seeing the Forest for the Trees: A Large Scale, Continuously Updating Meta-Analysis of Frontier LLMs](http://arxiv.org/abs/2502.18791v1), 2025-02-26

(90) [Active Few-Shot Learning for Text Classification](http://arxiv.org/abs/2502.18782v1), 2025-02-26

(91) [Towards Optimal Multi-draft Speculative Decoding](http://arxiv.org/abs/2502.18779v1), 2025-02-26

(92) [M2-omni: Advancing Omni-MLLM for Comprehensive Modality Support with Competitive Performance](http://arxiv.org/abs/2502.18778v1), 2025-02-26

(93) [Plutus: Benchmarking Large Language Models in Low-Resource Greek Finance](http://arxiv.org/abs/2502.18772v1), 2025-02-26

(94) [Exploring Graph Tasks with Pure LLMs: A Comprehensive Benchmark and Investigation](http://arxiv.org/abs/2502.18771v1), 2025-02-26

(95) [Reward Shaping to Mitigate Reward Hacking in RLHF](http://arxiv.org/abs/2502.18770v1), 2025-02-26

(96) [CommGPT: A Graph and Retrieval-Augmented Multimodal Communication Foundation Model](http://arxiv.org/abs/2502.18763v1), 2025-02-26

(97) [Training Large Recommendation Models via Graph-Language Token Alignment](http://dx.doi.org/10.1145/3701716.3715583), 2025-02-26

(98) [M-ANT: Efficient Low-bit Group Quantization for LLMs via Mathematically Adaptive Numerical Type](http://arxiv.org/abs/2502.18755v1), 2025-02-26

(99) [AgentSociety Challenge: Designing LLM Agents for User Modeling and Recommendation on Web Platforms](http://arxiv.org/abs/2502.18754v1), 2025-02-26

(100) [Automatic Prompt Optimization via Heuristic Search: A Survey](http://arxiv.org/abs/2502.18746v1), 2025-02-26
